package.json

```
@@ -39,7 +39,8 @@
     "test:watch": "jest --watchAll --coverage",
     "lint": "tslint -c tslint.json -p tsconfig.json \"src/**/*.ts\"",
     "prettier": "prettier --write \"{src,templates,test}/**/*.{ts,js}\"",
-    "prettier:check": "prettier --check \"{src,templates,test}/**/*.{ts,js}\""
+    "prettier:check": "prettier --check \"{src,templates,test}/**/*.{ts,js}\"",
```

Replace these with single quotes to make it extra super clear that the wildcards are being parsed by prettier, not the shell

```suggestion
    "prettier:check": "prettier --check '{src,templates,test}/**/*.{ts,js}'",
```